docs/software/vpn.md

```diff
@@ -225,4 +226,72 @@ YouTube video tutorial (German language): `Raspberry Pi & PiVPN mit WireGuard: I
 
 <iframe src="https://www.youtube-nocookie.com/embed/yRkdzGmnvA4?rel=0" frameborder="0" allow="fullscreen" width="560" height="315" loading="lazy"></iframe>
 
+## Tailscale
+
+Zero config VPN.
+
+Tailscale is a VPN service that makes the devices and applications you own accessible anywhere in the world, securely and effortlessly. It enables encrypted point-to-point connections using the open source WireGuard protocol, which means only devices on your private network can communicate with each other.
+
+![Tailscale logo](../assets/images/tailscale-logo.svg){: width="242" height="44" loading="lazy"}
+
+=== "Quick start"
+
+    Step 1: Sign up for an account
+
+    [Sign up for a Tailscale account](https://login.tailscale.com/start). Get started with a free personal plan or trial for an organizational plan.
+
+    Tailscale requires a Single Sign-On (SSO) provider, so you’ll need a Google, Microsoft, GitHub, Okta, OneLogin, or other supported SSO identity provider account to begin.
+
+    Step 2: Add a machine to your network
+
+    ```sh
+    tailscale up
+    ```
+
+    Tailscale helps you connect your devices together. For that to be possible, Tailscale needs to run on your device.
```

Comment:
Tailscale is already being installed here, do you mean other devices that should be part of the network? In that case, maybe:
```
For that to be possible, Tailscale needs to also be installed on other devices that you want to connect to.
```